src/Tribe/Main.php

```diff
@@ -401,6 +398,9 @@ public function bootstrap() {
 	public function bind_implementations() {
 		tribe_singleton( 'tickets.main', $this );
 
+		// Initialize the Service Provider for Tickets.
+		tribe_register_provider( Tribe__Tickets__Service_Provider::class );
+
```

Comment:
Should the `@since` be updated for the `bind_implementations` and `bootstrap` method?

Reply:
Not in this case, no major changes to the existence of the method signature of the method/class.